tests/test_write_anything.py

```diff
@@ -40,30 +43,40 @@
     "SPDXJsonExample2.2.json-tag",
 }
 
+
 @pytest.mark.parametrize("out_format", ['rdf', 'yaml', 'xml', 'json', 'tag'])
-@pytest.mark.parametrize("in_file", test_files, ids=lambda x: os.path.basename(x))
-def test_write_anything(in_file, out_format, tmpdir):
-    in_basename = os.path.basename(in_file)
-    if in_basename == "SPDXSBOMExample.spdx.yml" or in_basename == "SPDXSBOMExample.tag" or in_basename == "SPDXJsonExample2.2.json":
+@pytest.mark.parametrize("in_file_json_yaml_xml", test_files_json_yaml_xml, ids=lambda x: os.path.basename(x))
+def test_write_anything_json_yaml_xml(in_file_json_yaml_xml, out_format, tmpdir):
+    in_basename = os.path.basename(in_file_json_yaml_xml)
+    if in_basename == "SPDXSBOMExample.spdx.yml" or in_basename == "SPDXJsonExample2.2.json":
         # conversion of spdx2.2 is not yet done
         return
-    doc, error = parse_anything.parse_file(in_file)
+    write_anything_test(in_basename, in_file_json_yaml_xml, out_format, tmpdir)
+
+
+@pytest.mark.parametrize("out_format_rdf_tag", ['rdf', 'tag'])
+@pytest.mark.parametrize("in_file_rdf_tag", test_files_rdf_tag, ids=lambda x: os.path.basename(x))
```

Comment:
Same here: `out_format` and `in_file` is enough

Reply:
done